newflasher.c

```diff
@@ -4754,18 +4754,31 @@ int main(int argc, char *argv[])
 
 	printf("\n");
 	printf("Processing boot delivery...\n\n");
+	char bootzip[256];
```

Comment:
Better initialize with char bootzip[256]{0};

Reply:
any specific reason ? as other variables are also initialised in the same way

Reply:
It's best pratice nowadays, but we can leave it I would not fight for it